client/extensions/woocommerce/app/order/order-details.js

```diff
@@ -66,32 +119,34 @@ class OrderDetails extends Component {
 					<div className="order__details-totals">
 						<div className="order__details-total-discount">
 							<div className="order__details-totals-label">{ translate( 'Discount' ) }</div>
-							<div className="order__details-totals-value">{ order.discount_total }</div>
+							<div className="order__details-totals-value">
+								{ formatCurrency( order.discount_total, order.currency ) || order.discount_total }
+							</div>
 						</div>
 						<div className="order__details-total-shipping">
 							<div className="order__details-totals-label">{ translate( 'Shipping' ) }</div>
-							<div className="order__details-totals-value">{ order.shipping_total }</div>
+							<div className="order__details-totals-value">
+								{ formatCurrency( order.shipping_total, order.currency ) || order.shipping_total }
+							</div>
 						</div>
 						<div className="order__details-total">
 							<div className="order__details-totals-label">{ translate( 'Total' ) }</div>
-							<div className="order__details-totals-value">{ order.total }</div>
+							<div className="order__details-totals-value">
+								{ formatCurrency( order.total, order.currency ) || order.total }
+							</div>
 						</div>
+						{ refundValue
+							? <div className="order__details-total-refund">
+								<div className="order__details-totals-label">{ translate( 'Refunded' ) }</div>
```

Comment:
Hi! I've found a possible matching string that has already been translated **37** times:
`translate( 'Refunded To' )` *ES Score: 12*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/woo-order-product-display)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).